nilearn/plotting/_utils.py

```diff
@@ -44,8 +44,8 @@ def check_surface_plotting_inputs(
     if isinstance(surf_mesh, SurfaceImage):
         raise TypeError(
             "'surf_mesh' cannot be a SurfaceImage instance. ",
-            "Accepted types are: str, list of two numpy.ndarray, Mesh, "
-            "PolyMesh, or None.",
+            "Accepted types are: str, list of two numpy.ndarray, "
```

Comment:
I think that in the long term we will also deprecate the 2 numpy arrays as input and only allow InMemoryMesh and PolyMesh, no?

But not here because that would break the API.

Reply:
Yes, I agree.